### PR TITLE
Rbac/find all role

### PR DESCRIPTION
**Pull request by petersg83**
⚠️  Wait merge of #6264 
(once #6264 is merged, to have a correct diff, you'll probably need to change the base or the PR so it redo the diff)

### CODE REVIEW

File: packages/strapi-admin/controllers/Role.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict';
```
Comment (Convly) [inline packages/strapi-admin/controllers/Role.js:1]: New controllers name should be in `kebab-case`

File: packages/strapi-admin/services/role.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  create(attributes) {
+    return strapi.query('role', 'admin').create(attributes);
+  },
+
+  /**
+   * fetch a role in database
+   * @param params query params to find the role
+   * @returns {Promise<user>}
+   */
+  fetch(params) {
```
Comment (Convly) [inline packages/strapi-admin/services/role.js:16]: Maybe consider renaming to `findOne` & `find` to stay consistent with the query layer and the other services? (or perhaps this is a new convention?)

File: packages/strapi-admin/services/role.js [hunk 3]
```diff
@@ -12,11 +12,21 @@ const create = attributes => {
...
  */
-const findOne = params => {
+const findOne = (params = {}) => {
   return strapi.query('role', 'admin').findOne(params);
 };
 
+/**
+ * Find roles in database
+ * @param params query params to find the roles
+ * @returns {Promise<role>}
```
Comment (alexandrebodin) [inline packages/strapi-admin/services/role.js:22]: Pormise<Array<role>>

File: packages/strapi-admin/controllers/role.js [hunk 4]
```diff
@@ -13,4 +13,10 @@ module.exports = {
       data: role,
     };
   },
+  async find(ctx) {
+    const roles = await strapi.admin.services.role.find({ _limit: -1 });
```
Comment (alexandrebodin) [inline packages/strapi-admin/controllers/role.js:17]: We could create a findAll service so the controller staus as dumb as possible

Comment (petersg83) [reply packages/strapi-admin/controllers/role.js:17]: Yes I agree

**Pull request merged**